### ISSUE TITLE
[Bug]: console.time() bug in Chrome with Live Preview

### ISSUE THREAD

**Issue by leosilberg** [wallabyjs/console-ninja#286]
### What happened?

Console Ninja isn't displaying the time log and below is the error in Chrome

![Document - Brave 10_04_2024 10_15_06](https://github.com/wallabyjs/console-ninja/assets/25746648/5344b944-8fa4-481d-b206-f8d03ebacddc)

I'm not sure if this a problem with Live Preview or Console Ninja that's consuming the log call  as explained in this issue https://stackoverflow.com/a/50524698

### Version

 v1.0.301 

### Steps to reproduce (or sample repo)

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <script>
      console.clear();
      console.time("Time");
      for (let i = 0; i <= 1000; i++) {
        console.log(i);
      }
      console.timeEnd("Time");
    </script>
  </body>
</html>
```

### Log output

```shell
09:59:44.870 info host starting tools discovery and integration
09:59:44.883 info host looking for tools to patch
09:59:44.917 info host found 1 tool file(s) to patch
09:59:44.919 info host patching c:\Users\...\.vscode\extensions\ms-vscode.live-server-0.4.13\out\extension.js
09:59:44.925 info host Host's startup and patch time: 712 ms
09:59:44.931 info host completed tools discovery and integration
09:59:44.951 info host worker Starting Host Worker...
09:59:48.172 info host client connected: buildHook { address: '127.0.0.1', family: 'IPv4', port: 55405 }
09:59:48.130 info buildHook-9820 installing build hook for live-preview-extension at c:\users\...\appdata\local\programs\microsoft vs code, node v18.18.2
09:59:48.158 info buildHook-9820 allowed tools: vite, angular, jest, webpack, next.js, nest.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, live-preview-extension, node
09:59:48.158 info buildHook-9820 running tools: live-preview-extension
09:59:48.161 info buildHook-9820 installing fs interceptor
09:59:48.188 info buildHook-9820 file processed (stat): 2 log points, error handler: false
09:59:48.193 info buildHook-9820 file processed (stat): 1 log points, error handler: false
09:59:48.196 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.202 info buildHook-9820 file processed (stat): 2 log points, error handler: false
09:59:48.207 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.211 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.214 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.217 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.221 info buildHook-9820 file processed (stat): 2 log points, error handler: false
09:59:48.223 info buildHook-9820 file processed (stat): 1 log points, error handler: false
09:59:48.225 info buildHook-9820 file processed (stat): 1 log points, error handler: false
09:59:48.227 info buildHook-9820 file processed (stat): 1 log points, error handler: false
09:59:48.229 info buildHook-9820 file processed (stat): 3 log points, error handler: false
09:59:48.233 info buildHook-9820 file processed (stat): 7 log points, error handler: false
09:59:48.235 info buildHook-9820 file processed (stat): 4 log points, error handler: false
09:59:48.237 info buildHook-9820 file processed (stat): 0 log points, error handler: false
09:59:48.237 info buildHook-9820 allowed tools: vite, angular, jest, webpack, next.js, nest.js, cypress, http-server, serve, live-server, nuxt, remix, qwik, hydrogen, serverless, astro, live-preview-extension, node
09:59:48.237 info buildHook-9820 running tools: live-preview-extension
09:59:48.541 info host client connected: runtimeHook { address: '127.0.0.1', family: 'IPv4', port: 55405 }
10:10:13.598 info buildHook-9820 file processed (memory): 0 log points, error handler: false
10:10:13.612 info host client disconnected: runtimeHook 1001
10:10:13.612 info host client disconnected: runtimeHook
10:10:15.528 info buildHook-9820 file processed (stat): 0 log points, error handler: false
10:10:15.538 info buildHook-9820 file processed (memory): 4 log points, error handler: false
10:10:15.905 info host client connected: runtimeHook { address: '127.0.0.1', family: 'IPv4', port: 55405 }
10:10:16.062 info buildHook-9820 file processed (stat): 0 log points, error handler: false
10:10:17.193 info buildHook-9820 file processed (memory): 4 log points, error handler: false
10:10:17.206 info host client disconnected: runtimeHook 1001
10:10:17.206 info host client disconnected: runtimeHook
10:10:17.382 info buildHook-9820 file processed (stat): 0 log points, error handler: false
10:10:17.495 info host client connected: runtimeHook { address: '127.0.0.1', family: 'IPv4', port: 55405 }
10:10:27.720 info buildHook-9820 file processed (stat): 4 log points, error handler: false
10:10:27.730 info host client disconnected: runtimeHook 1001
10:10:27.730 info host client disconnected: runtimeHook
10:10:28.032 info host client connected: runtimeHook { address: '127.0.0.1', family: 'IPv4', port: 55405 }
```

**Comment by smcenlly** [comment]
Thanks for reporting the problem. We're working on a fix.

**Comment by smcenlly** [comment]
This is now fixed in Console Ninja `v1.0.304`.
